cellxgene_schema_cli/cellxgene_schema/write_labels.py

```diff
@@ -271,7 +278,15 @@ def _get_labels(
         column_definition = self._flatten_column_def_with_dependencies(column_definition)
 
         if label_type == "curie":
-            if "curie_constraints" not in column_definition:
+            curie_constraints = None
+
+            # Prefer using curie_constraints defined on add_labels, if available
```

Comment:
i updated how we fetch the curie constraints for adding labels because by default, it uses the generic column-level curie constraints, which for `cell_line_ontology_term_id` only allows "na". for `cell_line_ontology_term_id`, we actually want the curie constraints set by the dependency rule, for when `cell_type_ontology_term_id` is a special CL term we validate that `cell_line_ontology_term_id` is a valid EFO descendant.

there could be a better way to design this, like naming the curie constraints somehow and re-using them more easily. but i think this was fine for an experimental project.